stripe-ui-core/res/values/totranslate.xml

```diff
@@ -1,4 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
     <string name="stripe_paymentsheet_address_element_same_as_shipping">Billing address is same as shipping</string>
+
+    <string name="expiration_date_hint">MM / YY</string>
```

Comment:
Do we have a process to translate these? Should we do it as part of this PR?

Reply:
I will add this in a later PR